test/index.js

```diff
@@ -9,7 +9,7 @@ const queryElement = require('./util/queryElement');
 const queryElements = require('./util/queryElements');
 
 let session, geckodriver;
-const firefoxArgs = process.env.CI ? ['-headless'] : [];
+const firefoxArgs = process.env.DEBUG ? [] : ['-headless'];
```

Comment:
It's not documented yet, but the OP says to change the wiki when merging.

(I think it would be better to have the documentation in the tree, so PRs like this one can update the documentation in the PR, and people can review the documentation changes...)

Reply:
Right, it would be flipping the current instructions around `CI=1 npm run regression`